tests/spec/rainforest/joinByInvitation_logout_tryToLoginWith_NotAllowedEmail_InvalidPassword.rfml

```diff
@@ -7,7 +7,7 @@
 Click on the 'create a new team' link below the form where it says 'Welcome! Enter your team's Koding domain.'
 Do you see 'Let's sign you up!' form with 'Email Address' and 'Team Name' text fields and a 'NEXT' button?
 
-Enter "rainforest+{{ random.email }}" in the 'Email Address' and "{{ random.last_name }}{{ random.number }}" in 'Team Name' fields and click on 'NEXT' button
+Enter "{{ random.email }}" in the 'Email Address' and "{{ random.last_name }}{{ random.number }}" in 'Team Name' fields and click on 'NEXT' button
```

Comment:
It's not that both emails are randomly being generated to be the same but that koding appears to only check prior to the `+` for uniqueness

https://github.com/koding/koding/blob/ba0fa3efcbcb26eb1ce7aa5612dc405db37e32a8/workers/social/lib/social/models/user/emailsanitize.coffee#L18
https://github.com/koding/koding/blob/ba0fa3efcbcb26eb1ce7aa5612dc405db37e32a8/workers/social/lib/social/models/user/emailsanitize.test.coffee#L38

Admittedly I'm not overly familiar with the codebase so those functions may not apply here but the behavior seems to fit.